packetbeat/_meta/fields.yml

```diff
@@ -1,2083 +0,0 @@
-- key: common
```

Comment:
Seems like this was a left over file that should already have been deleted before?

Reply:
Yes. It is a duplication of `packetbeat/_meta/fields.common.yml` with fields which coming from `protos`. Nothing used this file at all.